lib/env-options.js

```diff
@@ -0,0 +1,13 @@
+/* @flow */
+
+const re = /^dotenv_config_(encoding|path|debug)$/
+
+module.exports = function optionMatcher () {
+  return Object.entries(process.env).reduce(function (acc, [key, val]) {
```

Comment:
Because we know exactly which 3 keys we want, this can be a simple object. May make testing easier to keep in a separate file

Reply:
I was trying to be consistent with the lib/cli-options.js which uses a reducer but it is simpler to just use a plain object.

Reply:
changes ready for review. Note I could change the case sensitivity [as mentioned here](https://github.com/motdotla/dotenv/issues/351#issuecomment-443817081) if you'd like.